src/services/itemMembership/repository.ts

```diff
@@ -776,48 +777,49 @@ export class ItemMembershipRepository extends MutableRepository<
       ORDER BY account_id, nlevel(item_path), permission;
     `)) as MoveHousekeepingType[];
 
-    const changes: ResultMoveHousekeeping = {
-      inserts: [],
-      deletes: [],
-    };
-
-    rows.reduce((chngs, row) => {
-      const {
-        accountId,
-        itemPath,
-        permission: p,
-        action,
-        inherited: ip,
-        action2IgnoreInherited,
-      } = row;
-
-      if (action === 0) {
-        return chngs;
-      }
-      if (action === 2 && action2IgnoreInherited) {
-        return chngs;
-      }
-
-      const permission = p;
-      const inherited = ip;
-
-      // permission (inherited) at the "origin" better than inherited one at "destination"
-      if (action === 1 && PermissionLevelCompare.gt(permission, inherited)) {
-        chngs.inserts.push({
+    const changes = rows.reduce<ResultMoveHousekeeping>(
+      (chngs, row) => {
+        const {
           accountId,
           itemPath,
-          permission,
-          creatorId: accountId,
-        });
-      }
+          permission: p,
+          action,
+          inherited: ip,
+          action2IgnoreInherited,
+        } = row;
+
+        if (action === 0) {
+          return chngs;
+        }
+        if (action === 2 && action2IgnoreInherited) {
+          return chngs;
+        }
 
-      // permission worse or equal to inherited one at "destination"
-      if (action === 2 && PermissionLevelCompare.lte(permission, inherited)) {
-        chngs.deletes.push({ accountId, itemPath });
-      }
+        const permission = p;
+        const inherited = ip;
```

Comment:
Why it is not possible to use the `permission` and `inherited` directly from the `row`?
```suggestion
```

Reply:
I have no f***!ng idea 🤷